src/translator_sv.h

```diff
@@ -70,16 +71,26 @@ Problem!
 * Uppdaterat den till senaste versionen 1.8.9.1
 2015/09/12
 * Fixat lite särksirvningar och inkonsekvenser
+2020/01/08
+* Uppdaterat den till senaste språkversionen 1.8.15
+
+Bytte ut Deprecated från Föråldrad till Obsolet
+
+VHDL översättningarna är kanske inte perfekta, då jag endast använt de en gång tidigare.
+     lämnade use clause orörd, då jag inte hittade en lämplig översättning för den.
+
```

Comment:
Would be nice to have an English translation of the update as well.

Reply:
I can fix that, pusing an update soon.